taxi_service/urls.py

```diff
@@ -14,8 +14,10 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
+
 
 urlpatterns = [
     path("admin/", admin.site.urls),
+    path("", include("taxi.urls", namespace="taxi"))
 ]
```

Comment:
You didn't follow the instruction in the `Serving static files during development` section. Your code will not work in production
https://docs.djangoproject.com/en/4.1/howto/static-files/

Reply:
Oh, I see. Usually, static files don't work without this step. We always added it. I reread the documentation, and it sounds really like it's not obligatory. So, If everything works you can omit this step.

Reply:
Tnx for response 🙂